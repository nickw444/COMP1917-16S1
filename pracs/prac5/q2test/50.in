50
7 -5 8 10 0 1 4 1 -4 0 2 0 6 -10 -10 -4 -4 -8 0 8 7 10 -8 7 6 0 10 -5 3 4 -7 0 5 -2 -4 -10 -6 -1 3 -1 -2 -2 8 -1 2 -3 5 6 -4 -6
