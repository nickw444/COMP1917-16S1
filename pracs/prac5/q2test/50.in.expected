average of array: 0.320000
